doc/whatsnew.md

```diff
@@ -1,4 +1,4 @@
-# Novel Auxiliary Options
+# What's New
```

Comment:
This file is messy, we need to clean it up

Reply:
First of all, do we need it? If we do, do we need both section " Interpolation with tabulated pair potentials" and "Type embedding"?

Reply:
We need to put what's new in v2.0, but not in the current form.

Reply:
OK.